docs/quickstarts/twitter.rst

```diff
@@ -54,8 +54,51 @@ run:
     $ export OAUTHLIB_INSECURE_TRANSPORT=1
     $ python twitter.py
 
-Visit http://localhost:5000 in your browser, and you should start the OAuth dance
-immediately.
+Visit ``http://127.0.0.1:5000`` in your browser, and you should start the OAuth dance
+immediately after you click on the Signin with Twitter hyperlink.
+
+In your view, make sure you use a plain hyperlink to begin the dance
+so that the redirect loads Twitter's confirmation page in the browser.
```

Comment:
This sentence is full of jargon, and hard to understand if you are new to OAuth. Which view are you referring to? How is a "plain hyperlink" different from other options? By "dance", do you mean OAuth or something else? Which redirect are you referring to? What if I don't want to load Twitter's confirmation page in the browser?

Reply:
yes I'll re-write this and attempt to clean up the jargon - good point about possibly not wanting to load the confirmation page in the browser - I didn't consider that.